bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-toolbar/toolbar-submenu-item/component.jsx

```diff
@@ -50,18 +50,22 @@ export default class ToolbarSubmenuItem extends Component {
   }
 
   render() {
+
+    const { className, customIcon, icon, label } = this.props
+
     return (
       <div className={styles.buttonWrapper}>
         <Button
           hideLabel
           role="button"
-          color="default"
-          size="md"
-          label={this.props.label}
-          icon={this.props.icon}
-          customIcon={this.props.customIcon}
-          onMouseUp={this.handleOnMouseUp}
-          className={this.props.className}
+          color={'default'}
+          size={'md'}
+          label={label}
+          aria-label={label}
+          icon={icon}
+          customIcon={customIcon}
+          onClick={this.handleItemClick}
```

Comment:
`onMouseUp` was intentional, we need it to support a text shape on iOS devices.
.. and `handleItemClick` doesn't exist in this file.

Change it back, please.